Data warning

awolfe76 opened the pull request:
Closes #1009 

I didn't want to use an `Alert` because I didn't want it taking up too much space above the input. I don't think it has to dominate the page.

![data-warning](https://user-images.githubusercontent.com/2932572/35881757-e3ca3c62-0b4f-11e8-9a7d-ec820d780170.png)

awolfe76 committed:
remove out of place message from institutions page

awolfe76 committed:
add warning about uploading PII data

wpears commented on src/institutions/Header.jsx:
```diff
@@ -18,10 +18,6 @@ const InstitutionsHeader = ({ filingPeriod }) => {
           You may file HMDA data for your authorized institutions below.
         </p>
-        <p>
-          Your progress will be saved if you leave the platform before
```
Is there some guidance on removing this language?

awolfe76 replied:
Sorry. I should have created an issue for this a while ago.

That language isn't useful in this location. It's much more useful during the upload process, where we also have it for large files. Here it just adds more noise.

I had talked to @designlanguage a while back about it and then forgot to create the issue, but he agreed.

Then, in other more recent conversations with @jmarin, it came up that it might not be a good idea to say this up front because we __need__ filers to allow at least the upload to finish. When we show it during the upload process the actual upload is already complete.

awolfe76 commented:
@wpears - hold of on merging this one. The PR on the back-end for this edit was merged so I'm going to finish this up with that work.

awolfe76 committed:
Merge branch 'master' into data-warning

awolfe76 committed:
highlight q666; also update caption styles to be more readable (have better spacing)

awolfe76 commented:
_Removing WIP_

## Initial Warning

I've updated the warning on the upload, just a slight color change. I thought it should match the highlight for Q666 (more in a minute). 

![upload-warning](https://user-images.githubusercontent.com/2932572/35984082-9f059374-0cc1-11e8-9f4a-b65bf03aca50.png)

## Q666

I've highlighted Q666, just using a simple background color (warning) and some padding. But, I've also updated the `<caption>` and `.caption` a bit to improve the spacing. (I think there's more to do here but this seems like a good first step.)

I've left Q666 at the bottom of the page, rather than moving it to the top. I think it's actually more noticeable closer to the verification checkbox. But, I'm open to suggestions on styling to highlight it. I'm not sure if using the same color as the "your file has edits" warning is a good idea ... maybe it's fine. Thoughts? @designlanguage @wpears

### Full screenshot showing updated captions and highlighted Q666

![quality-edits](https://user-images.githubusercontent.com/2932572/35984164-d44db6a6-0cc1-11e8-8d8d-253cad390ac1.png)

wpears commented:
I like the changes, I'd be interested to hear George's thoughts on the same-color warning though.

awolfe76 committed:
Merge branch 'master' into data-warning

designlanguage commented:
@wpears - The same coloring is tricky. We've established red to mean that something is wrong and it can't proceed until fixed. This seems more like a yellow alert.

The downside to having the same color is that it could seem like repeated information. If this edit being highlighted because it needs more attention than the others, can it be included in the first alert on the page? As is, that would be a lot of content, but would it help for a filer to see it right away?

awolfe76 committed:
don't render Q66 text

awolfe76 committed:
fix class vs className

awolfe76 commented:
@wpears - b5ab7a9 should finish this one off.

@designlanguage - I think based on the conversation earlier you are ok with this as is, right?
